### DISCUSSION TITLE
Web Pythonanywhere Error

### DISCUSSION THREAD

**Discussion by nabilaazarin24** [teamalgoritma/community#912] [Q&A] [answered]
Selamat siang, saya Nabila Azarin dari kelas ZEUS
Ingin bertanya mengenai error bot pythonanywhere yang telah dibuat yang telah dilampirkan.
<img width="924" alt="image" src="https://github.com/user-attachments/assets/e7306d78-d50a-43a6-b4e9-165ffdc5a95b">
<img width="591" alt="image" src="https://github.com/user-attachments/assets/a968ecdf-e436-481f-aacf-9324590e8c96">
Bagaimana utk pembetulannya? terima kasih
https://www.pythonanywhere.com/user/nabilaazarin24/files/var/log/nabilaazarin24.pythonanywhere.com.error.log

**Comment by dwiknrd** [comment, marked as answer]
Dear Ibu [nabilaazarin24](https://github.com/nabilaazarin24),

Terima kasih telah menghubungi kami. 

Dari error yang ibu lampirkan kami hanya dapat melihat dari screenshot yang ibu lampirkan. Pastikan bahwa Ibu sudah menambahkan kode `import sys` sebelum pendefinisian path file. Selain itu, pada bagian path, silahkan hapus tanda "/" yang ada dibagian akhir. 

Semoga dapat membantu. Apabila ada pertanyaan lebih lanjut silahkan ajukan dengan membalas comment ini.

Best Regards,
Team Algoritma

**Reply by nabilaazarin24** [reply]
Terima kasih untuk perbaikannya. Untuk arahan sudah saya lakukan, tetapi masih error dan tidak jalan utk app nya
<img width="721" alt="image" src="https://github.com/user-attachments/assets/1791ba83-00ed-4e01-8975-5bce22ea1980">
<img width="1354" alt="image" src="https://github.com/user-attachments/assets/37b95aac-3819-42cc-9e5c-276bb9fe48f9">
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/fe8fc919-aa02-4979-9c1a-ee546c0b45a5">
<img width="998" alt="image" src="https://github.com/user-attachments/assets/a8191c4a-d01c-4a72-995b-a466a6f9496d">
